resources/skins.citizen.styles/components/Footer.less

```diff
@@ -82,23 +84,27 @@
 		flex-grow: 1;
 		align-items: flex-end;
 
-		@media ( max-width: @max-width-breakpoint-tablet ) {
```

Comment:
The negative margin is intended to keep the text aligned with the footer, we should keep that